scripts/git_divergence/divergence.py

```diff
@@ -2,14 +2,25 @@
 import os
 import re
 import sys
+import argparse
 
 from git import Repo
 
 MASTER = os.getenv("GIT_AUTH_BRANCH", default="master")
 
 DATETIME = "%Y-%m-%d %H:%M:%S%z"
 
-BRANCH_FILTER = r"(.*RC\/.*)|([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)"
+ap = argparse.ArgumentParser()
+ap.add_argument("-p", "--pattern", required=False, help="regex pattern for BRANCH_FILTER")
+
+args = vars(ap.parse_args())
+
+if args["pattern"] is not None:
+    # A filter patern was passed in
+    pattern = re.compile(args["pattern"])
+    BRANCH_FILTER = pattern
```

Comment:
Is `filter` misleading here? This somewhat reads to me as only return b ranches in the filter vs ignoring them.

Might be a good idea to make this a bit clearer? `branch_ignore` or `branch_mask`?

Reply:
Ya I like that idea! Now that we can link to the code too :partying_face: